tests/test_resource_tracker.py

```diff
@@ -284,3 +284,43 @@ def test_resource_tracker_sigterm(self):
     def test_resource_tracker_sigkill(self):
         # Uncatchable signal.
         self.check_resource_tracker_death(signal.SIGKILL, True)
+
+    @pytest.mark.skipIf(sys.version_info < (3, 8),
```

Comment:
Not `skipIf`, pytest complains, just `skipif`. All lowercase.

Reply:
Thanks - weird that `pytest` did not error out on this.